client/app.js

```diff
@@ -12,6 +12,8 @@ window.moment = require('moment');
 window.sprintf = require('sprintf-js').sprintf;
 
 // Vendor libraries, order matters
+window.c3 = require('c3/c3.js');
```

Comment:
Patternfly expects C3 to be global like this.  I will look into moving it into our global file though